src/components/Transaction.js

```diff
@@ -16,21 +17,22 @@ export default function Transaction({
   type,
   notes,
   tag,
+  deleteTransaction,
 }) {
+  const [isNotesSectionVisible, setIsNotesSectionVisible] = useState(false)
```

Comment:
Wouldn't it be better to name your state like following?
```suggestion
  const [notesSectionVisible, setNotesSectionVisible] = useState(false)
```